tendermint/src/public_key.rs

```diff
@@ -1,6 +1,7 @@
 //! Public keys used in Tendermint networks
 
 pub use ed25519_dalek::PublicKey as Ed25519;
+mod pub_key_request;
 
```

Comment:
I guess it'd be good to expose `PubKeyRequest`?
```suggestion
pub use pub_key_request::PubKeyRequest;
```

Reply:
Nice catch! Late night coding strikes again.